cpp/arcticdb/processing/operation_types.hpp

```diff
@@ -520,3 +520,182 @@ bool operator()(T t, const ankerl::unordered_dense::set<U>& u) const {
+        return fmt::format_to(ctx.out(), "*");
+    }
+};
+
+template<>
+struct formatter<arcticdb::DivideOperator> {
+    template<typename ParseContext>
+    constexpr auto parse(ParseContext &ctx) { return ctx.begin(); }
+
+    template<typename FormatContext>
+    constexpr auto format(const arcticdb::DivideOperator&, FormatContext &ctx) const {
+        return fmt::format_to(ctx.out(), "/");
+    }
+};
+
+template<>
+struct formatter<arcticdb::EqualsOperator> {
+    template<typename ParseContext>
+    constexpr auto parse(ParseContext &ctx) { return ctx.begin(); }
+
+    template<typename FormatContext>
+    constexpr auto format(const arcticdb::EqualsOperator&, FormatContext &ctx) const {
+        return fmt::format_to(ctx.out(), "=");
+    }
+};
+
+template<>
+struct formatter<arcticdb::NotEqualsOperator> {
+    template<typename ParseContext>
+    constexpr auto parse(ParseContext &ctx) { return ctx.begin(); }
+
+    template<typename FormatContext>
+    constexpr auto format(const arcticdb::NotEqualsOperator&, FormatContext &ctx) const {
+        return fmt::format_to(ctx.out(), "!=");
+    }
+};
+
+template<>
+struct formatter<arcticdb::LessThanOperator> {
+    template<typename ParseContext>
+    constexpr auto parse(ParseContext &ctx) { return ctx.begin(); }
+
+    template<typename FormatContext>
+    constexpr auto format(const arcticdb::LessThanOperator&, FormatContext &ctx) const {
+        return fmt::format_to(ctx.out(), "<");
+    }
+};
+
+template<>
+struct formatter<arcticdb::LessThanEqualsOperator> {
+    template<typename ParseContext>
+    constexpr auto parse(ParseContext &ctx) { return ctx.begin(); }
+
+    template<typename FormatContext>
+    constexpr auto format(const arcticdb::LessThanEqualsOperator&, FormatContext &ctx) const {
+        return fmt::format_to(ctx.out(), "<=");
+    }
+};
+
+template<>
+struct formatter<arcticdb::GreaterThanOperator> {
+    template<typename ParseContext>
+    constexpr auto parse(ParseContext &ctx) { return ctx.begin(); }
+
+    template<typename FormatContext>
+    constexpr auto format(const arcticdb::GreaterThanOperator&, FormatContext &ctx) const {
+        return fmt::format_to(ctx.out(), ">");
+    }
+};
+
+template<>
+struct formatter<arcticdb::GreaterThanEqualsOperator> {
+    template<typename ParseContext>
+    constexpr auto parse(ParseContext &ctx) { return ctx.begin(); }
+
+    template<typename FormatContext>
+    constexpr auto format(const arcticdb::GreaterThanEqualsOperator&, FormatContext &ctx) const {
+        return fmt::format_to(ctx.out(), ">=");
+    }
+};
+
+template<>
+struct formatter<arcticdb::IsInOperator> {
+    template<typename ParseContext>
+    constexpr auto parse(ParseContext &ctx) { return ctx.begin(); }
+
+    template<typename FormatContext>
+    constexpr auto format(const arcticdb::IsInOperator&, FormatContext &ctx) const {
+        return fmt::format_to(ctx.out(), "IN");
```

Comment:
`IS IN`

Reply:
Fixed in [a907db9](https://github.com/man-group/ArcticDB/pull/1557/commits/a907db94c4366371434d048c630f796fcbc7a60e)